templates/web/pages/product/product_page.tt.html

```diff
@@ -148,7 +148,7 @@ <h2 class="title-1" property="food:name" itemprop="name">[% title %]</h1>
             <!-- owner -->
             [% IF owner %]
                 <p>
-                    [% FILTER format(lang("sources_manufacturer")) %]<a href="facets/editors/[% owner %]">[% owner_org.name %]</a>[% END %]
+                    [% FILTER format(lang("sources_manufacturer")) %]<a href="/facets/editors/[% owner %]">[% owner_org.name %]</a>[% END %]
```

Comment:
The fixup to add the root to the URL path looks good; one extra idea: to enable localization, I think we might be able to use:

```suggestion
                    [% FILTER format(lang("sources_manufacturer")) %]<a href="/facets[% canonicalize_taxonomy_tag_link("editors", owner) %]">[% owner_org.name %]</a>[% END %]
```

Idea based on a pattern noticed at: https://github.com/openfoodfacts/openfoodfacts-server/blob/ebc8ab252c45013e96214e4cb77d011a7c05bcd7/templates/web/common/site_layout.tt.html#L168

Reply:
Thanks for the PR @RayanGoubet , can you test @jayaddison 's suggestion and commit it? Thanks!

Reply:
I'll try that !

Reply:
Thank you @RayanGoubet!